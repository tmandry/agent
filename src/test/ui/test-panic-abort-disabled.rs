// no-prefer-dynamic
// compile-flags: --test -Cpanic=abort
// run-flags: --test-threads=1
// run-fail
// check-run-results

#![cfg(test)]

#[test]
fn it_works() {
    assert_eq!(1 + 1, 2);
}

#[test]
#[should_panic]
fn it_panics() {
    assert_eq!(1 + 1, 4);
}
