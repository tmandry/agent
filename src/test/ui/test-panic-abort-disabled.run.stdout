
running 2 tests
test it_panics ... 